### PR TITLE
[script.video.randomtv] 1.1.0

### PR DESCRIPTION
**Pull request by gmccauley**
### Description
v1.1.0 (28 April 2017)
  - Added Auto Stop Feature
  - Replaced xbmc.sleep with xbmc.Monitor().waitForAbort

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.video.randomtv/addon.py [hunk 1]
```diff
@@ -167,11 +177,33 @@ def onPlayBackStopped(self):
 
 	# Start Player
 	player.play(item=myPlaylist)
-	xbmc.sleep(100)
 #
 
 
-while (not xbmc.Monitor().abortRequested()):
+while (not xbmc.Monitor().waitForAbort(0.1)):
```
Comment (Razzeee) [inline script.video.randomtv/addon.py]: Is it really necessary to do it every 0.1? Might slow down some low powered devices. Higher is better here as long as your addons responsiveness doesn't change from it.

Comment (gmccauley) [inline script.video.randomtv/addon.py]: I'll try higher.  I know I tried 1 second in the last version that used xbmc.sleep at the end of the loop and it was a too laggy for my taste.

Comment (gmccauley) [inline script.video.randomtv/addon.py]: Oh, and it does seem to run fine on both a Pi2 and Pi3.

**Comment by gmccauley** [comment]
Thanks Razzeee, I tried 1 second and it does seem fine.  I think the .1 second was from before I reworked the code to use a playlist.

**Pull request merged**